android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/view/ButtonProperties.kt

```diff
@@ -39,13 +40,15 @@ data class ButtonProperties(
   override val fillMaxHeight: Boolean = false,
   override val clickable: String = "false",
   override val visible: String = "true",
+  val contentColor: String? = null,
```

Comment:
Do we have a default content color?

Reply:
Yea will default to the color inferred from the `ServiceStatus`. The default status is UPCOMING; color code Grey.